chat-plugins/modlog.js

```diff
@@ -129,11 +129,11 @@ function checkRipgrepAvailability() {
 	return Config.ripgrepmodlog;
 }
 
-function runModlog(room, searchString, exactSearch, maxLines) {
+async function runModlog(room, searchString, exactSearch, maxLines) {
 	const useRipgrep = checkRipgrepAvailability();
 	let fileNameList = [];
 	if (room === 'all') {
-		const fileList = fs.readdirSync(`${__dirname}/${LOG_PATH}`);
+		const fileList = await FS(`${__dirname}/${LOG_PATH}`).readdir();
```

Comment:
...actually one of the benefits `FS` brings is that you don't need to do the whole `__dirname` thing anymore.